Kconfig.thingset

```
@@ -36,6 +36,21 @@ config THINGSET_NUM_JSON_TOKENS
 	  Trying to parse a request with more than the maximum number of tokens will result in an
 	  error.
 
+config THINGSET_BINARY_MAX_DEPTH
+	int "Max depth of hierarchy serialisable by the binary serialiser."
+	default 4
+	help
+	  Configure the maximum depth of a node hierarchy that can be serialised by
+	  the binary serialiser. The default of 4 is usually suitable for most use
+	  cases, but you may wish to increase this in certain scenarios.
+
+config THINGSET_REPORT_RECORD_SERIALISATION
+	bool "Enable full record serialisation in reports."
```

Please use AE to be consistent with the rest of the code.

```suggestion
config THINGSET_REPORT_RECORD_SERIALIZATION
	bool "Enable full record serialization in reports."
```